pkg/api/v1/types.go

```diff
@@ -1478,6 +1484,10 @@ type Container struct {
 	// Default is false.
 	// +optional
 	TTY bool `json:"tty,omitempty" protobuf:"varint,18,opt,name=tty"`
+	// List of environment variables to set in the container.
```

Comment:
1. Add validation of keys.
2. Add the precedence ordering which should be order of precedence Env, EnvFrom, Services.
3. Add support for expansion of Envs with a runtimeVal.  I noticed there weren't (m)any tests for it.